students/jungmin/westagram/westagram/settings.py

```diff
@@ -0,0 +1,149 @@
+"""
+Django settings for westagram project.
+
+Generated by 'django-admin startproject' using Django 4.0.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+from my_settings import DATABASES, SECRET_KEY
```

Comment:
```suggestion
from pathlib import Path

from my_settings import DATABASES, SECRET_KEY
```
import 하는 모듈의 구분을 위해 한줄 띄워주세요~